src/govuk/components/accordion/accordion.js

```diff
@@ -50,7 +50,6 @@ Accordion.prototype.init = function () {
   }
```

Comment:
As moving the summary inside the button is a slightly unexpected change (which then requires the change in this commit), we discussed that it would be good to 
- move code that moves the summary inside the button into this commit to have a self contained commit of all the related changes
- add a bit more detail to the commit message about why we’re moving the summary inside the button (mainly to do with design as we didn’t want the summary text below the button element)
- why moving the summary means that it can no longer be a span

Reply:
A small thing but it seems that this comment from https://github.com/alphagov/govuk-frontend/pull/2257/commits/408b8cdd85e7abcd2a2152f14e130d1cad263815

> (@nnagewad) "We’ve decided to move the summary in order to better visually illustrate the additional context of accordion heading by placing it directly beneath the heading as opposed to placing it directly beneath the reveal/expand button"[4]

should probably be in https://github.com/alphagov/govuk-frontend/pull/2257/commits/61020eb0de84361ff2a2b9da3dc0ca53d106b223. 

We could also signpost to https://html.spec.whatwg.org/multipage/form-elements.html#the-button-element re: 
> Content model: 
> [Phrasing content](https://html.spec.whatwg.org/multipage/dom.html#phrasing-content-2)

Reply:
Moved the justification into the other commit & added the additional sign post to phrasing.